src/components/Nav/Nav.js

```diff
@@ -2,8 +2,156 @@ import React, { Component } from 'react';
 import './Nav.scss';
 
 class Nav extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isToggleOn: false,
+    };
+  }
+
+  handleOnMouse = () => {
+    const { isToggleOn } = this.state;
+
+    this.setState({
+      isToggleOn: !isToggleOn,
+    });
+  };
+
+  handleOffMouse = () => {
+    const { isToggleOn } = this.state;
+
+    this.setState({
+      isToggleOn: !isToggleOn,
+    });
+  };
+
   render() {
-    return <div></div>;
+    const { isToggleOn } = this.state;
+    const { handleOnMouse, handleOffMouse } = this;
+    return (
```

Comment:
```suggestion

    return (
```
- 구분되는 로직 끼리는 한줄 띄워주세요
- 함수까지 구조분해 할당하는건 좀 과한거 같아서 이 부분은 그냥 this로 써주세요

Reply:
네 리뷰 반영하여 this로 변경했습니다.